source/agg-pipeline-builder/export-pipeline-results.txt

```diff
@@ -0,0 +1,64 @@
+.. _export-agg-builder-results:
+
+===================================
+Export Aggregation Pipeline Results
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To use your aggregation data in other external tools, you can export your 
```

Comment:
Note: I'm not completely sure if this would be the primary use case for exporting, but it's the one that came to mind for me. Thoughts?

Reply:
I think this makes sense as a use case. I just might tweak "aggregation data" to "aggregated data".

One other possible use case to mention might be to export pipeline results and then import those results to a different collection. (Note: I'm not sure how useful this is, given that users can accomplish the same thing with the $out stage.)

Reply:
Gotcha. In that case, I think I'll stick to this.